src/ResourceManager/Sql/Commands.Sql/Database Backup/Services/AzureSqlDatabaseBackupCommunicator.cs

```diff
@@ -179,14 +188,105 @@ public Management.Sql.LegacySdk.Models.DeletedDatabaseBackup GetDeletedDatabaseB
             string baPolicyName, 
             DatabaseBackupLongTermRetentionPolicyCreateOrUpdateParameters parameters)
         {
-            return GetCurrentSqlClient().DatabaseBackup.CreateOrUpdateDatabaseBackupLongTermRetentionPolicy(
+            return GetLegacySqlClient().DatabaseBackup.CreateOrUpdateDatabaseBackupLongTermRetentionPolicy(
                 resourceGroupName, 
                 serverName, 
                 databaseName, 
                 baPolicyName, 
                 parameters).DatabaseBackupLongTermRetentionPolicy;
         }
 
+        /// <summary>
+        /// Gets a database's Long Term Retention policy.
+        /// </summary>
+        /// <param name="resourceGroup">The resource group name.</param>
+        /// <param name="serverName">The server name.</param>
+        /// <param name="databaseName">The database name.</param>
+        public Management.Sql.Models.LongTermRetentionPolicy GetDatabaseLongTermRetentionPolicy(
+            string resourceGroup,
+            string serverName,
+            string databaseName)
+        {
+            return GetCurrentSqlClient().LongTermRetentionPolicies.Get(resourceGroup, serverName, databaseName);
```

Comment:
How long will you need this "Current" vs "Legacy" syntax? Is that a permanent change?

Reply:
No, not permanent. The Vaults are being decommissioned after about a month, so not long.